README.md

```diff
@@ -41,6 +41,9 @@ git checkout <tag>
 
 2. Build a wheel using `poetry build`.
 
+> [!NOTE]
+> You may need to add read privaleges to your *Users/\<your-username>/.pyenv/* folder.
```

Comment:
Looks good! Total nitpick, but can we spell it "privileges" just so no one gets confused about if we're referencing a python library or anything else. Also, is there a reason there's a backslash (`\`) before `<your-username>`?

Reply:
If you take a look at the [README on the branch](https://github.com/mongodb/mut/tree/install-note#modern-mut-v010-and-later) you can see that the backslash doesn't show up - it's just a necessary escape character within the brackets.

Reply:
Ah whoops, great thanks!